GPflowOpt/acquisition/mes.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Joachim van der Herten, Nicolas Knudde
```

Comment:
How are we going to do the copyright. Per file or just point to the contributors file? At some point many people will have touched the same files.